src/test/java/com/auth0/AuthenticationControllerTest.java

```diff
@@ -505,4 +505,39 @@ public void shouldCheckSessionFallbackWhenHandleCalledWithRequest() throws Excep
         controller.handle(request);
     }
 
+    @Test
+    public void shouldAllowOrganizationParameter() {
+        AuthenticationController controller = AuthenticationController.newBuilder("DOMAIN", "CLIENT_ID", "SECRET")
+                .withOrganization("orgId_abc123")
+                .build();
+
+        String authUrl = controller.buildAuthorizeUrl(new MockHttpServletRequest(), new MockHttpServletResponse(), "https://me.com/redirect")
+                .build();
+        assertThat(authUrl, containsString("organization=orgId_abc123"));
+    }
+
+    @Test
+    public void shouldThrowOnNullOrganizationParameter() {
+        exception.expect(NullPointerException.class);
```

Comment:
Not a fan of throwing a Runtime exception on invalid arguments (should be `InvalidArgumentException`), but it is consistent with the rest of the library.